xbmc/pvr/channels/PVRChannelsPath.cpp

```diff
@@ -81,7 +92,8 @@ CPVRChannelsPath::CPVRChannelsPath(const std::string& strPath)
 
         if (!m_clientID.empty() && m_iChannelUID >= 0)
         {
-          m_kind = Kind::CHANNEL; // pvr://channels/(tv|radio)/<groupname>/<addonid>_<channeluid>.pvr
+          m_kind = Kind::
+              CHANNEL; // pvr://channels/(tv|radio)/<groupname>/[<instanceid>@]<addonid>_<channeluid>.pvr
```

Comment:
You implemented the new path so that instanceid is always present. So, the comment should read: "// pvr://channels/(tv|radio)/<groupname>/<instanceid>@<addonid>_<channeluid>.pvr"